modules/crm_core_match/src/Controller/MatcherController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\crm_core_match\Controller\MatcherController.
+ */
+
+namespace Drupal\crm_core_match\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\crm_core_match\Matcher\MatcherConfigInterface;
+
+/**
+ * Class MatcherController.
+ */
+class MatcherController extends ControllerBase {
+
+  /**
+   * Gets the edit matcher title.
+   *
+   * @param \Drupal\crm_core_match\Matcher\MatcherConfigInterface $crm_core_match
+   *   The edited matcher.
+   *
+   * @return string
+   *   The page title.
+   */
+  public function editTitle(MatcherConfigInterface $crm_core_match) {
```

Comment:
If you don't already, make sure you follow https://www.drupal.org/node/2350509. That aims to provide default routes and controllers for stuff like this.